vortexasdk/utils.py

```diff
@@ -225,3 +227,17 @@ def sts_param_value(param):
         return {"exclude": True, "x_filter": False}
     else:
         return {"exclude": False, "x_filter": False}
+
+
+class PAGINATION_STRATEGIES(Enum):
+    OFFSET = "OFFSET"
+    SEARCH_AFTER = "SEARCH_AFTER"
+
+
+def showDeprecatedGeoExclusionRulesWarning(
+    disable_geographic_exclusion_rules: Optional[bool], logger: Logger
```

Comment:
nit: should we default it to `False` and type it as `bool` rather than having it as Optional?